apps/src/code-studio/pd/form_components/button_list.jsx

```diff
@@ -110,6 +139,7 @@ const ButtonList = React.createClass({
   }
 });
 
+export default ButtonList;
```

Comment:
Seems weird to be exporting ButtonList twice to me but this isn't that big a deal

Reply:
Agreed. But I'd argue it's more weird that it was being exported by name instead of default before, and I didn't want to break anything that was using it the old way.

Reply:
However I could also remove the named export and update all the places its imported?